### ISSUE TITLE
Determining Non-Functional Requirements

### ISSUE THREAD

**Issue by onurkomurcu** [bounswe/bounswe2022group2#21]
## Description

As we have learned in the lesson, we should firstly determine the requirements of the project. It's the initial step before starting to work further. There are different types of requirements and we decided to collect them under distinct titles. Besides, we also decided to assign different people for each type of requirements and a person who is not assigned to that title's issue to review it. We all will read each requirements issue and we are open to comment further, add new requirements to those titles even we are not assigned to them.

Our base titles for the requirements for now are as follows:
- Functional Requirements
    * User Requirements
    * System Requirements
- Non-Functional Requirements

This structure may be enhanced/enlarged and new base titles can be added in the future. If you can find, you can write them as a  comment to this issue.

According to our decision in meeting 2, responsible people for the **user requirements** are:
- [x] @onurkomurcu [Security and Performance and Reliability requirements](https://github.com/bounswe/bounswe2022group2/issues/33)
- [x] @hasancan-code  [Usability & Disaster Recovery requirements](https://github.com/bounswe/bounswe2022group2/issues/34)
- [x] @egemenatikk - [Availability and Accessibility And Privacy requirements](https://github.com/bounswe/bounswe2022group2/issues/30)

These people will create separate issue for the subtitle s/he work and will link those issues under this main issue with a summary comment. We will manage all user requirement related things here with the help of hyperlinks, comments and so on.

There is a one reviewer for this issue and she will review what the assignees found, what others are commented at the end and will make the last sentences before closing the issue to be sure that everything works well.

**Deadline: 13.03.2022 - Sunday - 20:00**
**Reviewer: @codingAku** - Deadline: 14.03.2022 - Monday - 23:59

**Comment by egemenatikk** [comment]
**INFO:**
I will be detailing the items for the **Availability and Accessibility and Privacy** as subtitles of non-functional requirements.
A separate issue is created for this purpose. You can view it in issue #30 .
Now, this issue depends on that sub-issue to be reviewed and closed.

**Comment by onurkomurcu** [comment]
**INFO:**
I will be detailing the items for the **Security and Performance and Reliability** as subtitles of non-functional requirements.
A separate issue is created for this purpose. You can view it in issue #33 .
Now, this issue depends on that sub-issue to be reviewed and closed.

**Comment by codingAku** [comment]
I believe it has been an adequate entrance to the non-functional requirements. Feel free to re-open for any additions. Thank you to all assignees for their hard work. Issue closed.
